enzyme/Enzyme/EnzymeLogic.cpp

```diff
@@ -2894,7 +2896,6 @@ const AugmentedReturn &EnzymeLogic::CreateAugmentedPrimal(
   for (auto user : fnusers) {
     if (removeStruct) {
       IRBuilder<> B(user);
-      auto n = user->getName().str();
       user->setName("");
       SmallVector<Value *, 4> args(user->arg_begin(), user->arg_end());
       auto rep = B.CreateCall(NewF, args);
```

Comment:
@wsmoses this call here should get the name of `user` right?

Reply:
Yeah that would be nice

We can use takeName to do